os/android/iahwc2.cpp

```diff
@@ -230,19 +230,34 @@ static inline void supported(char const *func) {
 HWC2::Error IAHWC2::CreateVirtualDisplay(uint32_t width, uint32_t height,
                                          int32_t *format,
                                          hwc2_display_t *display) {
```

Comment:
Ensure *format and *display are not NULL.

Reply:
Okay, will update.